### PR TITLE
Fixes #34259 - add slot for rex dropdown in host page

### PR DESCRIPTION
**Pull request by amirfefer**

### CODE REVIEW

**Comment by amirfefer** [comment]
for https://github.com/theforeman/foreman_remote_execution/pull/679

**Comment by stejskalleos** [comment]
:heavy_check_mark: LGTM, works as expected:

Screenshot from 2022-01-20 09-36-11

File: webpack/assets/javascripts/react_app/components/HostDetails/ActionsBar/index.js [hunk 1]
```diff
@@ -137,20 +140,29 @@ const ActionsBar = ({
...
-        alignments={{ default: 'right' }}
-        toggle={<KebabToggle id="hostdetails-kebab" onToggle={onKebabToggle} />}
-        isOpen={kebabIsOpen}
-        isPlain
-        dropdownItems={dropdownItems.concat(registeredItems)}
-      />
+      <Split hasGutter>
+        <SplitItem>
+          <Slot hostId={hostId} id="_rex-host-features" />
+        </SplitItem>
```
Comment (ezr-ondrej) [inline webpack/assets/javascripts/react_app/components/HostDetails/ActionsBar/index.js:146]: Cant that be a slot for more then one action? Like follows:
```suggestion
        <Slot hostId={hostId} id="host-toolbar-actions" />
```

**Review by ezr-ondrej — APPROVED** [review]
Thanks @amirfefer I agree we can hold off the proper Slot for all plugins until we have two plugins needing it :+1:

**Pull request merged**